tools/github_login.py

```diff
@@ -6,7 +6,7 @@
 import sys
 from getpass import getpass
 import github3
-from . config import read_config, write_config
+from config import read_config, write_config
```

Comment:
Why do you use a relative import without point?
If you have the error: `ValueError: Attempted relative import in non-package`
You can try with:
`python -m tools.tx_pull.py`